client/scss/utilities/_functions.scss

```diff
@@ -30,10 +30,17 @@
 }
 
 // Get the width of a number of columns at a given viewport width
-@function span-columns($viewport-size, $number) {
+@function span-columns($viewport-size, $number, $no-final-gutter: false) {
   $columns-at-size: map-get($total-columns, $viewport-size);
   $gutter-at-size: map-get($gutter-width, $viewport-size);
-  $total-gutter: $number * $gutter-at-size;
+  $total-gutter: null;
+
+  @if ($no-final-gutter) {
```

Comment:
Can we use sass's ternary:
```SASS
$total-gutter: if($no-final-gutter, ($number - 1) * $gutter-at-size, $number * $gutter-at-size);
```
Although looking at it - where are we sending this through as true?